src/app/scripts/cac/control/cac-control-sidebar-directions.js

```diff
@@ -9,6 +9,9 @@ CAC.Control.SidebarDirections = (function ($, Control, BikeOptions, Geocoder, Ma
 
     var METERS_PER_MILE = 1609.34;
 
+    // Number of millis to wait on input changes before sending directions request
+    var directionDebounceMillis = 750;
```

Comment:
Might make sense to either capitalize the added constants, or change `METERS_PER_MILE` to lower case, for consistency.